Headers/DebugServer2/Target/ProcessBase.h

```diff
@@ -141,9 +141,9 @@ class ProcessBase {
 
 public:
   virtual Architecture::GDBDescriptor const *
```

Comment:
I'm not sure. I think not. I'll mark it final (which will require virtual
too, but should not have perf impact).

On Thursday, April 28, 2016, strager <notifications@github.com> wrote:

> In Headers/DebugServer2/Target/ProcessBase.h
> <https://github.com/facebook/ds2/pull/304#discussion_r61447808>:
>
> > @@ -141,9 +141,9 @@ class ProcessBase {
> >
> >  public:
> >    virtual Architecture::GDBDescriptor const *
>
> Does this still need to be virtual?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/304/files/e7e1963afb287907b44f6accbe4a37ce9db0b078..ca0df3832dee3d8d1fa3898e79c93a7708702f5d#r61447808>
>


-- 
Stephane Sezer